From 95bd64a893e036ccc54ceef25d4c55fcfe3a4475 Mon Sep 17 00:00:00 2001
From: Ernesto Ramos <ernesto@ti.com>
Date: Wed, 28 Jul 2010 09:45:27 -0500
Subject: [PATCH 077/116] staging:ti dspbridge: remove DSP_SUCCEEDED macro from services

Since status succeeded is now 0 macro DSP_SUCCEEDED
is not necessary anymore.

Signed-off-by: Ernesto Ramos <ernesto@ti.com>
Signed-off-by: Greg Kroah-Hartman <gregkh@suse.de>
---
 drivers/staging/tidspbridge/services/cfg.c |   10 +++++-----
 1 files changed, 5 insertions(+), 5 deletions(-)

diff --git a/drivers/staging/tidspbridge/services/cfg.c b/drivers/staging/tidspbridge/services/cfg.c
index 4ab3cd7..c3f33f6 100644
--- a/drivers/staging/tidspbridge/services/cfg.c
+++ b/drivers/staging/tidspbridge/services/cfg.c
@@ -62,7 +62,7 @@ int cfg_get_auto_start(struct cfg_devnode *dev_node_obj,
 		status = -EFAULT;
 	if (!auto_start || !drv_datap)
 		status = -EFAULT;
-	if (DSP_SUCCEEDED(status))
+	if (!status)
 		*auto_start = (drv_datap->base_img) ? 1 : 0;
 
 	DBC_ENSURE((status == 0 &&
@@ -93,7 +93,7 @@ int cfg_get_dev_object(struct cfg_devnode *dev_node_obj,
 		status = -EFAULT;
 
 	dw_buf_size = sizeof(value);
-	if (DSP_SUCCEEDED(status)) {
+	if (!status) {
 
 		/* check the device string and then store dev object */
 		if (!
@@ -127,7 +127,7 @@ int cfg_get_exec_file(struct cfg_devnode *dev_node_obj, u32 buf_size,
 	if (strlen(drv_datap->base_img) > buf_size)
 		status = -EINVAL;
 
-	if (DSP_SUCCEEDED(status) && drv_datap->base_img)
+	if (!status && drv_datap->base_img)
 		strcpy(str_exec_file, drv_datap->base_img);
 
 	if (DSP_FAILED(status))
@@ -178,7 +178,7 @@ int cfg_get_object(u32 *value, u8 dw_type)
 		*value = 0;
 		pr_err("%s: Failed, status 0x%x\n", __func__, status);
 	}
-	DBC_ENSURE((DSP_SUCCEEDED(status) && *value != 0) ||
+	DBC_ENSURE((!status && *value != 0) ||
 		   (DSP_FAILED(status) && *value == 0));
 	return status;
 }
@@ -211,7 +211,7 @@ int cfg_set_dev_object(struct cfg_devnode *dev_node_obj, u32 value)
 	if (!dev_node_obj)
 		status = -EFAULT;
 
-	if (DSP_SUCCEEDED(status)) {
+	if (!status) {
 		/* Store the Bridge device object in the Registry */
 
 		if (!(strcmp((char *)dev_node_obj, "TIOMAP1510")))
-- 
1.7.1

